backend/api/job.proto

```diff
@@ -32,9 +32,9 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
     key: "default";
     value: {
       schema: {
-        json_schema: {
-          ref: ".api.Status";
-        }
+	json_schema: {
```

Comment:
Can you justify why this is the preferred indent?

Reply:
I may prefer the existing indent. Can't understand why send out this PR.
Is it due to PR author's IDE/editor setting?